scio-core/src/main/scala/com/spotify/scio/schemas/SchemaMaterializer.scala

```diff
@@ -150,7 +150,7 @@ object SchemaMaterializer {
     schema
       .toList(v)
       .asScala
-      .map[schema.schema.Repr](dispatchEncode(schema.schema, fieldType.getCollectionElementType))
+      .map(dispatchEncode(schema.schema, fieldType.getCollectionElementType): A => schema.schema.Repr)
```

Comment:
Is this something that Scala 3.0.0-RC2 fails to compile?

Reply:
Without the `[schema.schema.Repr]` Scala 3 fails to compile.
With it Scala 2 fails to compile because back in the 2.12 days, `map` had a second type parameter for `CanBuildFrom`.
This the cleanest solution to make the three version happy

Reply:
Ah, I see. Thank you!